src/index.html

```diff
@@ -1,12 +1,30 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Webpack Boilerplate</title>
+  <title>TO DO</title>
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
 </head>
+
 <body>
-    <h1>Hello Webpack</h1>
+  <div class="container">
+    <div class="item header">
+      <span class="material-icons list-icon">checklist</span>
```

Comment:
- [ ] Add the title "Today's To Do" as the original one. And once we are here, change the background color

Your style:

![image](https://user-images.githubusercontent.com/60631456/126236254-4fb5f94e-698d-4ab6-b1a4-2e5fb3c564a4.png)

Target style:

![image](https://user-images.githubusercontent.com/60631456/126236297-73b2a9e1-7c3e-4e23-b648-124957d710e7.png)